### PR TITLE
feat(react): New react module

### PR DESCRIPTION
**Pull request by ibgreen**
See #38 (added new tasks at bottom)
- Move generic react functionality out of the specialized modules and into a generic react module. 
- Move overlays and EditorModal to the generic react module
- Goal is to eventually remove the specific helpers, if feaible.

### CODE REVIEW

File: dev-docs/RFCs/editable-layers/generic-edit-mode.md [hunk 1]
```diff
@@ -115,21 +115,21 @@ class EditableGeoJsonLayer {
 
 ### Module layout
 
-We will need a `@nebula.gl/edit-modes` module separate from the `nebula.gl` module. The reason is because this new `@nebula/edit-modes` should have no deck.gl dependency.
+We will need a `@@deck.gl-community/editable-layers/edit-modes` module separate from the `@deck.gl-community/editable-layers` module. The reason is because this new `@nebula/edit-modes` should have no deck.gl dependency.
 
-- `nebula.gl`
-  - depends on `@nebula.gl/edit-modes`, `@nebula.gl/layers`, and all the other `@nebula/...` modules.
+- `@deck.gl-community/editable-layers`
+  - depends on `@@deck.gl-community/editable-layers/edit-modes`, `@@deck.gl-community/editable-layers/layers`, and all the other `@nebula/...` modules.
```
Comment (donmccurdy) [inline dev-docs/RFCs/editable-layers/generic-edit-mode.md]:
```suggestion
  - depends on `@deck.gl-community/editable-layers`
```

File: dev-docs/RFCs/editable-layers/generic-edit-mode.md [hunk 2]
```diff
@@ -115,21 +115,21 @@ class EditableGeoJsonLayer {
...
   - doesn't have much in it, just basically imports from the others and re-exports them
-- `@nebula.gl/edit-modes`
+- `@@deck.gl-community/editable-layers/edit-modes`
```
Comment (donmccurdy) [inline dev-docs/RFCs/editable-layers/generic-edit-mode.md]:
```suggestion
- `@deck.gl-community/editable-layers`
```

File: dev-docs/RFCs/editable-layers/generic-edit-mode.md [hunk 3]
```diff
@@ -115,21 +115,21 @@ class EditableGeoJsonLayer {
...
   - contains `EditMode` interface
   - contains other general purpose types and classes (e.g. event types like `ClickEvent`)
   - this module will be reused by `react-map-gl-draw`
-- `@nebula.gl/layers`
-  - depends on `@nebula.gl/edit-modes` and `deck.gl`
+- `@@deck.gl-community/editable-layers/layers`
+  - depends on `@@deck.gl-community/editable-layers/edit-modes` and `deck.gl`
   - contains `EditableGeoJsonLayer`, a deck.gl `CompositeLayer`
-- Other modules are unaffected (e.g. `@nebula.gl/overlays`)
+- Other modules are unaffected (e.g. `@@deck.gl-community/editable-layers/overlays`)
```
Comment (donmccurdy) [inline dev-docs/RFCs/editable-layers/generic-edit-mode.md:132]: See above. :)

File: dev-docs/RFCs/editable-layers/react-map-gl-draw.md [hunk 4]
```diff
@@ -5,7 +5,7 @@
 [react-map-gl](https://visgl.github.io/react-map-gl/) currently does not support drawing functions. However, we have got a couple of [users](https://github.com/uber/react-map-gl/issues/725) interested in this capability. Also it is one of P0 features on Kepler.gl 2019 [roadmap](https://github.com/uber/kepler.gl/wiki/Kepler.gl-2019-Roadmap#allow-drawing-on-map-to-create-paths-and-polygons--).
 
 Although [Mapbox/mapbox-gl-draw](https://github.com/mapbox/mapbox-gl-draw) provides drawing and editing features, because of its manipulating internal states, it cannot work well with React / Redux framework and therefore cannot be integrated with `react-map-gl`.
-Another `vis.gl` framework [Nebula.gl](http://nebula.gl) also provides geo editing functionality, but it heavily depends on [`deck.gl`](https://deck.gl/), which may not be suitable for non `deck.gl` use cases.
+Another `vis.gl` framework [Nebula.gl](http://@deck.gl-community/editable-layers) also provides geo editing functionality, but it heavily depends on [`deck.gl`](https://deck.gl/), which may not be suitable for non `deck.gl` use cases.
```
Comment (donmccurdy) [inline dev-docs/RFCs/editable-layers/react-map-gl-draw.md]:
```suggestion
Another `vis.gl` framework `@deck.gl-community/editable-layers` also provides geo editing functionality, but it heavily depends on [`deck.gl`](https://deck.gl/), which may not be suitable for non `deck.gl` use cases.
```

File: dev-docs/RFCs/editable-layers/react-map-gl-draw.md [hunk 5]
```diff
@@ -82,7 +82,7 @@ Returns is a map of [style objects](https://reactjs.org/docs/dom-elements.html#s
 
 ### Styling based on `state`:
 
-![img](https://raw.githubusercontent.com/uber-common/deck.gl-data/master/nebula.gl/edit-handle.png)
+![img](https://raw.githubusercontent.com/uber-common/deck.gl-data/master/@deck.gl-community/editable-layers/edit-handle.png)
```
Comment (donmccurdy) [inline dev-docs/RFCs/editable-layers/react-map-gl-draw.md]: Probably not the right URL?

**Review by donmccurdy — COMMENTED** [review]
Thank you for cleaning all this up! Unfortunately a RegExp may have gone astray, several broken URLs. 😅

File: dev-docs/RFCs/editable-layers/generic-edit-mode.md (hunk 3 above)
Comment (ibgreen) [reply dev-docs/RFCs/editable-layers/generic-edit-mode.md:132]: Yes, I think we should restore these RFCs from the original nebula repo, or remove them.

**Pull request merged**